### PR TITLE
Add Github workflow to run tests on PRs

### PR DESCRIPTION
**Pull request by fernando-nubank**

### CODE REVIEW

**Review by dchelimsky — APPROVED** [review]
💯

File: project.clj [hunk 1]
```diff
@@ -14,7 +14,8 @@
 
   :profiles {:dev {:plugins [[lein-cljsbuild "1.1.7"]
                              [lein-doo "0.1.11"]]
-                   :dependencies [[org.clojure/clojurescript "1.10.520"]]}}
+                   :dependencies [[org.clojure/clojurescript "1.10.520"]
+                                  [org.clojure/core.rrb-vector "0.1.1"]]}}
```
Comment (rafaelzlisboa) [inline project.clj:18]: why is that new dependency added?

**Review by rafaelzlisboa — APPROVED** [review]

File: project.clj (hunk 1 above)
Comment (dchelimsky) [reply project.clj:18]: A conflicting version of rrb-vector was being pulled in transitively by another dep.

Comment (fernando-nubank) [reply project.clj:18]: originally we got this transitively but that version has a bug the blocks us from running CLJS tests. I pinned it here to fix this

**Pull request merged**